src/nano_id/random.cljc

```diff
@@ -6,7 +6,10 @@
 
 (def ^:private secure-random
   #?(:clj  NanoID/secureRandom
-     :cljs js/crypto))
+     :cljs (try
+             js/crypto
+             (catch :default _
+               js/msCrypto))))
```

Comment:
```suggestion
     :cljs (or js/window.crypto js/window.msCrypto)))
```

I've tested this on a [fork](https://github.com/jkxyz/nano-id/commit/1aba51b56d32ba9f8e05c66978cd1f915b8c5497) with advanced optimizations, and it works correctly in Firefox/Chrome/IE11. For now we're using a polyfill, but it would be great to have this merged.

Reply:
Sorry for late response. As I said above, I'm not going to add IE11 support to the library itself. If you need it, please add it locally as described [here](https://github.com/zelark/nano-id#ie).